test/model/storage/ConsumerStorageTest.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoOauth\test\model\storage;
+
+use common_exception_NotFound;
+use common_persistence_KeyValuePersistence;
+use core_kernel_classes_Class;
+use core_kernel_classes_Property;
+use core_kernel_classes_Resource;
+use League\OAuth2\Client\Token\AccessToken;
+use oat\generis\model\data\Ontology;
+use oat\generis\persistence\PersistenceManager;
+use oat\generis\test\TestCase;
```

Comment:
This TestCase class is deprecated if I remember and we can just change it to \PHPUnit\Framework\TestCase.